VisualizationBase/src/items/ViewItem.h

```diff
@@ -172,9 +176,12 @@ class VISUALIZATIONBASE_API ViewItem : public Super<DeclarativeItem<ViewItem>> {
 		void ensureColumnExists(int column);
 
 		QVector<QVector<Model::Node*>> nodesGetter();
+
+		bool zoomLabelsEnabled_{};
```

Comment:
instead of a default initializer use `{true}` and remove the redundant initialization in the constructor.